gulp/_default.js

```diff
@@ -0,0 +1,30 @@
+const gulp = require('gulp')
+const theo = require('theo')
+const runSequence = require('run-sequence')
+
+// Universal Formats
+theo.registerFormat('json', `[
+ {{~#each props as |prop|}}
```

Comment:
Not a blocker, but it would be great to keep to the same ESLint guidelines we set up in the previous repo. We can always do this again post-PR.